### ISSUE TITLE
Flow-cli panic when deploy contract with argument of dictionaries type.

### ISSUE THREAD

**Issue by liuxiaobleach** [onflow/flow-cli#443]
When use flow-cli to deploy a contract with init func param type as Dictionaries.
For example:
pub contract Vault {
    pub var signers: {String: UInt256}
    init(data: {String: UInt256}) {
        ....
    }
}

And when I deploy this contract with flow-cli and local flow.json with deployment config.
It always panic.
I am sure my argument  is correct in flow.json.

So can you help me with this problem? However, if my config is wrong, please show a correct one.
Thank you~

**Comment by sideninja** [comment]
Can you share the part of the config defining the deploy argumetns?

**Comment by sideninja** [comment]
I've reproduced the issue and I'm working on it.

**Comment by liuxiaobleach** [comment]
Thank you for your replay.
My deployment args in flow.json:
"deployments": {
		"emulator": {
			"emulator-account": [
				{
					"name": "Vault",
					"args": [
						{
							"type": "Dictionary",
							"value": [
								{
									"key": "222",
									"value": "22"
								}
							]
						}
					]
				}
			]
		  }
	}

**Comment by sideninja** [comment]
Yeah @liuxiaobleach this is starting to look like a bit harder problem to solve due to how cadence decodes the dictionary type from JSON. An issue has been opened here https://github.com/onflow/cadence/issues/1378
I might think about a workaround until that is solved but can't make promises.

**Comment by liuxiaobleach** [comment]
Thank you.

**Comment by liuxiaobleach** [comment]
@sideninja Double check, is this fix already deployed on testnet now?

**Comment by sideninja** [comment]
Unfortunately, not yet, it's a bigger Cadence change and it might take some time. I will keep an eye on this.

**Comment by liuxiaobleach** [comment]
Ok, if the fix is deployed on flow testnet, please tell me~
